style.css

```diff
@@ -0,0 +1,153 @@
+.ul {
+  list-style: none;
+}
+
+.text-section {
+  background-image: url('img/Headerbg.jpg');
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] Great job with how far you've progressed with your project :100: I noticed that you completely removed the margin, but the previous reviewer suggested you reduce it. It is making your design a little different from the Figma design. Please add a margin of at least 1 rem to this class to solve this issue :+1: 
- [ ] Also from the image below you can see that the text in your paragraph with class `box p1` doesn't have the same font style as what was used in the Figma design, there are also some spacing issues that need to be fixed as well :+1: 
- [ ] Lastly, please make sure that there is some spacing between your `h1` element and your `p` element so that your design looks like an exact replica of the Figma design :100: Please refer to the images below :wink: 
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/212171872-52131d77-386b-404d-85b3-5520848c18dd.png)

- Your design
![image](https://user-images.githubusercontent.com/102699958/212171636-6db138dd-03be-493e-a5a9-7b6b02eb4137.png)